### ISSUE TITLE
Link quebrado no Fale conosco

### ISSUE THREAD

**Issue by ElidiaVidal** [redelivre/novo-cnpc#4]
- [ ] Logo após o "Acesse Formulário", no canto inferior direito da página, temos a opção:
"Para assistência ao usuário envie um e-mail para: laboratoriodeculturadigital@gmail.com" - o link do e-mail está quebrado.

**Comment by thalitasejanes** [comment]
ahn legal, na verdade eu acho que o botão seria o linque para acessar o formulario (o form enviaria o e-mail, precisa clarear isso), o fale conosco é com o cnpc, e o do laboratorio para suporte, precisa clarear este texto.

**Comment by ElidiaVidal** [comment]
Exato. O texto está confuso mesmo. A impressão que fica é: enviar e-mail clicando no "acesse formulário" para começar um diálogo; para assistência enviar e-mail.
E quando passamos o mouse em cima do e-mail também aparece a dica visual de um hiperlink, que no caso não tem.

**Comment by ElidiaVidal** [comment]
Sugestão de melhoria:

- [ ] Deixar apenas um botão de contato "Fale Conosco";

- [ ]  Dentro do formulário incluir mais um campo select para "tipo" (assistência técnica, diálogo e sugestões)
